src/corner_kick/storybook/webpack.config.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Webpack configuration file for Storybook
+ */
+
+const path = require('path');
+
+module.exports = (baseConfig, env, config) => {
+    // We support Typescript
+    config.module.rules.push({
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: [
+            {
+                loader: require.resolve('ts-loader'),
+            },
+        ],
+    });
+    config.resolve.extensions.push('.js', '.ts', '.tsx', '.json');
```

Comment:
Just as a note, it seems like we're doing this for the same file types in a few places now. Could they instead be unified?

Reply:
Done, I unified everything under `webpack.base.config.js` in the config folder and had the various additional configuration files import from there.

Reply:
Awesome!